spec/reports/github_api_client_post_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "sinatra/base"
+require "webmock/rspec"
+require "reports/github_api_client"
+
+class FakeGitHub < Sinatra::Base
+  attr_reader :gists
+
+  def initialize
+    super
+    @gists = []
+  end
+
+  post '/gists' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.